### PR TITLE
[#81] Implement player details sticky scroll

### PR DESCRIPTION
**Pull request by Tumne**
Completed sticky scroll for `card.js`

Screen Recording 2020-05-23 at 1 11 32 AM

### CODE REVIEW

File: src/components/players/details/placeholder/index.module.scss [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 .container {
     min-width: 400px;
+    padding-top: 5px;
```
Comment (kammanz) [inline src/components/players/details/placeholder/index.module.scss:5]: nice catch

File: src/components/players/details/card/index.module.scss [hunk 2]
```diff
@@ -2,60 +2,99 @@
...
+
+.playerDetails {
+  color: $grey2;
+  font-size: 14px;
+}
+
+.playerNumber {
+  flex: 1;
+  padding-right: 5px;
+  color: $background-color-purple;
```
Comment (kammanz) [inline src/components/players/details/card/index.module.scss]: We dont need this code. I'll do a hotfix.

**Review by kammanz — APPROVED** [review]
Slick work. I manually removed an extraneous line of code in `details/card/index.module.scss` at line 96 that set `color: $background-color-purple;`.

File: src/components/players/details/card/index.js [hunk 3]
```diff
@@ -1,41 +1,58 @@
...
+    weight_lbs,
+    teamColor,
+  },
+  playerTeamId,
+  isSticky,
+  isAnimated
+}) => (
+  <>
+    <div className={styles.card}>
+      <div className={styles.imageContainer}>
```
Comment (Tumne) [inline src/components/players/details/card/index.js:27]: Wack! Try again

File: src/components/players/details/card/index.js [hunk 4]
```diff
@@ -1,41 +1,58 @@
...
+          alt='player headshot'
+          onError={e => e.target.src = placeholderImg}
+        />
+      </div>
+    </div>
+    <div className={classnames(styles.card, styles.cardWrapper, isSticky && styles.borderBottom)}>
+      <div style={{ borderColor: teamColor }} className={styles.imageLine} />
+      <div className={styles.detailsContainer}>
+        <div className={styles.infoContainer}>
+          <div className={classnames(styles.thumb, isAnimated && styles.transition, isSticky && styles.animate)}>
```
Comment (Tumne) [inline src/components/players/details/card/index.js:39]: nice

**Review by Tumne — COMMENTED** [review]
Testing

**Pull request merged**